src/components/Indicator/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Container } from "./styled";
+//@ts-ignore
+import Bell from "./Bell";
+
+
+function Indicator({ onClick }) {
+  return (
+    <Container onClick={onClick}>
```

Comment:
if this is an interactive element lets make it a button so its accessible

Reply:
might be worth adding react-a11y to ensure we are accessible

Reply:
Adding to separate task